src/pages/sourcecast/Sourcecast.tsx

```diff
@@ -275,6 +283,26 @@ class Sourcecast extends React.Component<SourcecastProps> {
   }
 }
 
+const parseSourcecastUID = (props: SourcecastProps) => {
+  const pathname = props.location.pathname;
+  if (pathname.length > 11) {
+    const uid = pathname.substr(12);
+    if (uid !== props.uid && props.sourcecastIndex) {
+      props.sourcecastIndex
+        .filter(data => data.uid === uid)
```

Comment:
Find.

Communicate computational processes

Reply:
Thanks. Will fix it.